crates/sui-core/src/execution_cache/writeback_cache.rs

```diff
@@ -1060,9 +1153,7 @@ impl ObjectCacheRead for WritebackCache {
                 ObjectEntry::Deleted => (object_id, version, ObjectDigest::OBJECT_DIGEST_DELETED),
```

Comment:
Similar to above comment, can we still get `ObjectEntry::Deleted` or ObjectEntry::Wrapped` here?

Reply:
Cleaned this up by having a different enum type for the latest-object cache